spec/models/legalaid_application_spec.rb

```diff
@@ -16,4 +16,11 @@
   it 'should belong to an applicant' do
     expect(subject.applicant_id).to eq(applicant.id)
   end
+
+  describe 'Associations' do
+    it 'has many proceeding_types' do
```

Comment:
You should probably consider to use [this](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_record/association_matcher.rb) instead to do these sort of relation assertions.